client-templates/container-registry-agent/accept.json.sample

```diff
@@ -0,0 +1,48 @@
+{
+  "public": [
+  {
+    "method": "POST",
+      "path": "/api/v1/import/app"
+  },
+  {
+    "method": "POST",
+    "path": "/api/v2/import/done"
+  },
+  {
+    "method": "POST",
+    "path": "/api/v2/import/result"
+  }
+  ],
+  "private": [
+  {
+    "method": "GET",
+    "path": "/healthcheck",
```

Comment:
I'm not sure this is needed, as the registry will not send healthchecks to the remote DRA. wdyt?

Reply:
Not strictly necessary atm. But imho good to have this open from the get go, if we want to monitor it in the future.